radar.csv

```diff
@@ -1,64 +1,61 @@
+"Go",trial,"Languages and Frameworks",true,"<a href='https://golang.org/'>Go</a> is the natural language to use when extending Kubernetes with operators."
+"Java",trial,"Languages and Frameworks",true,""
+"Blazor.Net",assess,"Languages and Frameworks",true,""
+".NET Core",hold,"Languages and Frameworks",true,"Don't use .NET Core - we've moved on to .NET 5"
+"PowerShell",hold,"Languages and Frameworks",true,"Don't use PowerShell for new projects - prefer Bash"
+"WinForms",hold,"Languages and Frameworks",true,"Don't use WinForms for new projects - prefer Electron."
+"WPF",hold,"Languages and Frameworks",true,"Don't use WPF for new projects - prefer Electron"
+"Visual Studio",adopt,"Tools",true,""
+"TeamCity",adopt,"Tools",true,"TeamCity is our default build system"
+"GitHub",adopt,"Tools",true,"GitHub is our default version control system"
+"Application Insights",adopt,"Tools",true,""
+"Flyway",adopt,"Tools",true,""
+"Rider",adopt,"Tools",true,""
+"VS Code",adopt,"Tools",true,""
+"WSL 2",adopt,"Tools",true,"Windows Subsystem for Linux should be installed on every development machine"
+"CLIs",adopt,"Tools",true,""
+"RESTful APIs",adopt,"Tools",true,""
+"SignalR",adopt,"Tools",true,""
+"Hashicorp Vault",adopt,"Tools",true,""
+"Terraform",adopt,"Tools",true,""
+"Docker",adopt,"Tools",true,""
+"gRPC",trial,"Tools",true,""
+"Azure DevOps",trial,"Tools",true,"We believe Azure DevOps might be a fine alternative to TeamCity"
+"WhiteSource (vulnerability scanning)",trial,"Tools",true,""
+"DocFx",trial,"Tools",true,"<a href='https://dotnet.github.io/docfx/'>DocFX</a> is an extensible and scalable static documentation generator. We're experimenting with it for the <a href='https://progression-framework.red-gate.com'>web version of the progression framework</a>. Could this be a future solution to host our documentation too?"
+"GitHub Actions",assess,"Tools",true,"We're keeping an eye on GitHub actions as part of simplifying our build process."
+"Makefiles",assess,"Tools",true,""
+"Earthly",assess,"Tools",true,""
+"Feature Flagging vendors (Launch Darkly?)",assess,"Tools",true,"If your usage of feature flagging becomes sufficiently complicated then it's time to investigate third-party vendors."
+"Cake",assess,"Tools",true,""
+"Bazel",assess,"Tools",true,""
+"Raygun",hold,"Tools",true,"Deprecated in favour of Application Insights"
+"Virtualbox based Vagrant boxes",hold,"Tools",true,""
+"Keepass files in GitHub",hold,"Tools",true,""
+"Redgate.Build",hold,"Tools",true,""
+"Kubernetes",adopt,"Platforms",true,""
+"Docker",adopt,"Platforms",true,""
+"Electron",adopt,"Platforms",true,""
+"Backing store in SQL Server",adopt,"Platforms",true,""
+"Redgate Platfrom",adopt,"Platforms",true,"All new applications should be built on the Redgate Platform"
+"Linux x-64 (.NET supported versions)",adopt,"Platforms",true,""
+"Windows",adopt,"Platforms",true,""
+"Mac OSx",adopt,"Platforms",true,""
```

Comment:
Minor thing, but macOS is the term nowadays (yes, including the non-capitalised 'm' 😬)